src/test/scala/uk/gov/hmrc/test/api/cucumber/stepdefs/ifs/IFSInstalmentCalculationSteps.scala

```diff
@@ -692,22 +692,23 @@ class IFSInstalmentCalculationSteps extends ScalaDsl with EN with Eventually wit
 
     val expectedInstalmentCalculationResponse = InstalmentCalculationSummaryResponse(
       quoteDate,
-      10,
+      11,
```

Comment:
Do we know what happened for this to be previously passing and needing an update now?